builder/hyperv/common/step_mount_floppydrive.go

```diff
@@ -93,7 +94,8 @@ func (s *StepMountFloppydrive) Cleanup(state multistep.StateBag) {
 }
 
 func (s *StepMountFloppydrive) copyFloppy(path string) (string, error) {
-	tempdir, err := ioutil.TempDir("", "packer")
+	prefix, _ := configfile.ConfigTmpDir()
+	tempdir, err := ioutil.TempDir(prefix, "hyperv")
```

Comment:
the cut/paste of annoying to say the least. I like the idea of one-liner packer.TempDir() though since it would look like an "overload" of ioutil.TempDir. The env/config/tmp versions I don't understand their utility since you'd have to check if the env or config were defined and take remedial action.

So all those duo-lines would be reduced to:
`tempdir, err := packer.TempDir("hyperv")`

I should have thought of that...
As to determining prefix, is there a function that will return my current package name? Other languages have something like self.module_name().

Reply:
I initially thought `os.Args[0]` would be okay, but it only gives us `packer` when running a built binary; it does not when using `go run .`.
I think it's totally okay to hardcode `packer` somewhere; makes it simple 🙂.